packages/eslint-plugin-warp-drive/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "eslint-plugin-warp-drive",
   "description": "ESLint rules for Applications using WarpDrive or EmberData",
   "main": "./src/index.js",
-  "version": "0.0.3-alpha.37",
-  "private": true,
```

Comment:
I'd set this to private before because we didn't have new rules and it made it easier to adjust some tooling to not publish it